client/routes/workflow/helpers/summarize-events.js

```diff
@@ -114,15 +114,9 @@ const summaryExtractors = {
 
     return summary;
   },
-  WorkflowExecutionFailed: (d) => {
-    return { message: d.failure.message };
-  },
-  WorkflowTaskFailed: (d) => {
-    return { message: d.failure.message };
-  },
-  ChildWorkflowExecutionFailed: (d) => {
-    return { message: d.failure.message };
-  },
+  WorkflowExecutionFailed: (d) => ({ message: d.failure?.message }),
```

Comment:
You could probably define this lambda once and then just pass it.

Reply:
since it is one liner and there just 3 lines, i think it's better for readability as is